e2e/cypress/integration/tests/steppers.spec.js

```diff
@@ -0,0 +1,49 @@
+/// <reference types="cypress" />
+
+context('Increment steppers', () => {
+  beforeEach(() => {
+    cy.visit('/')
+  });
+
+  it('Increment regional population', () => {
+    cy.contains('Hospitalized Admissions peaks at 301');
+
+    cy.get('input.st-al').eq(0)
```

Comment:
Population is fairly "stable" despite changing today. Keep this and 15 market share too.